velox/exec/DistinctAggregations.cpp

```diff
@@ -185,6 +199,47 @@ class TypedDistinctAggregations : public DistinctAggregations {
     return input->template asUnchecked<RowVector>()->children();
   }
 
+  void extractForSpill(folly::Range<char**> groups, VectorPtr& result) const {
+    auto* arrayVector = result->as<ArrayVector>();
+    arrayVector->resize(groups.size());
```

Comment:
nit: perhaps, introduce temp variable numGroups to avoid repeating groups.size()